cdap-proto/src/main/java/co/cask/cdap/proto/ProtoConstraint.java

```diff
@@ -112,7 +118,16 @@ public long getMillisAfterTrigger() {
     }
 
     @Override
+    public void setWaitUntilMet(boolean waitUntilMet) {
+      super.setWaitUntilMet(waitUntilMet);
```

Comment:
no need to update super's `waitUntilMet` since we always require `waitUntilMet` to be true?